### ISSUE TITLE
Unsupported operator "ReduceSumSquare" // 不支持的运算符"ReduceSumSquare"

### ISSUE THREAD

**Issue by IskanderSh2** [rockchip-linux/rknn-toolkit2#253]
I am trying to convert my tensorflow keras NN to .rknn format from .onnx, but in convertaion process I get fatal error on layer "ReduceSumSquare", that isnt supports on rknn-toolkit2 v1.5.2
Will support for the "ReduceSumSquare" layer be added in the next version, I really need it

我正在尝试将我的tensorflow keras NN转换为。rknn格式从.onnx，但在转换过程中我在"ReduceSumSquare"层上得到一个致命错误，该错误在rknn-toolkit2版本1.5.2中不受支持
将在下一个版本中添加对"ReduceSumSquare"层的支持，我真的需要它

**Comment by StevenPod** [comment]
I completely agree, this problem has been around for a long time and I hope it will be fixed as soon as possible.
